sdk-base/src/main/java/com/mapbox/maps/extension/style/StyleInterface.kt

```diff
@@ -15,6 +15,12 @@ interface StyleInterface : StyleManagerInterface {
    */
   val pixelRatio: Float
 
+  /**
```

Comment:
We're 100% sure that interface is not implemented with any customer, right?

Reply:
yes, it's our abstractions. While for plugin we _potentially_ have a way to overwrite our default plugins - we do not have that for extensions and this interface is used with style extension only.
@pengdev @tobrun to confirm my understanding is correct here

Reply:
StyleInterface is implemented in the SDK module itself, we allow user to overwrite the plugin implementations, but the SDK module itself shouldn't and couldn't be implemented by the end users.

The cases I can think of user use the StyleInterface is that they could mock it in their tests.

Reply:
@pengdev sounds a bit strange to me still as we're operating with `Style` in our public API. Mostly it's one place:
```
/**
   * Callback to be invoked when a style has finished loading.
   */
  fun interface OnStyleLoaded {
    /**
     * Invoked when a style has finished loading.
     *
     * @param style the style that has finished loading
     */
    fun onStyleLoaded(style: Style)
  }
``` 
so for an end user guess it makes sense to mock `Style` then... In any case I'll drop instructions in PR description for that case.

Reply:
👍 didn't remember we return Style async instead of StyleInterface